core/lib/dal/src/protocol_versions_dal.rs

```diff
@@ -313,20 +322,86 @@ impl ProtocolVersionsDal<'_, '_> {
         })
     }
 
-    pub async fn last_version_id(&mut self) -> DalResult<Option<ProtocolVersionId>> {
-        Ok(sqlx::query!(
+    pub async fn get_patch_versions_for_vk(
```

Comment:
what should be the behavior when we query for 'old' minor protocols? should we return `vec![0]` then?

Reply:
yes, check migration, it adds patch 0 for all present protocol version, so the method will return vec![0]